language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -37,6 +38,31 @@ import DA.Daml.Project.Consts
 import DA.Daml.Project.Types
 import qualified DA.Daml.Project.Types as DATypes
 
+-- Referenced from 'writePackageJson'. Lifted here for easy
+-- maintenance.
+data ConfigConsts = ConfigConsts
+  { pkgDependencies :: [(T.Text, T.Text)]
```

Comment:
`[(T.Text, T.Text)]` doesn’t give much information to the reader. `HashMap NpmPackageName NpmPackageVersion` on the other hand would make it very clear and you can just call `toJSON` on it and throw away `fields`.